Makefile

```diff
@@ -42,6 +42,10 @@ clippy: pre-clippy
 	    -A unreadable_literal -A should_implement_trait -A verbose_bit_mask \
 	    -A implicit_hasher -A large_enum_variant -A new_without_default_derive \
 	    -A neg_cmp_op_on_partial_ord
+	cargo clippy --bench benches -- -A module_inception -A needless_pass_by_value -A cyclomatic_complexity \
```

Comment:
PTAL @breeswish

Reply:
Please check if there was anything missed.

Reply:
You only need to keep it identical to the list above.

Reply:
`--bench xx` means "Check all benches", so I think the previous line (cargo clippy without --bench) is not necessary any more. Also it seems that our integration tests are not clippy checked as well, so you may be want `--bins --tests --benches` as well.